Platforms/Docs/Common/debugging.md

```diff
@@ -29,6 +29,31 @@ This will scan for image headers between the specified addresses and load their
 symbols. More information on this command can be found in the Windbg help
 window.
 
+## Debugging Using GDB in VS Code
+
+For GCC builds, GDB can be used to debug instead. The symbols can be loaded by
+running `source MU_BASECORE/BaseTools/Scripts/efi_gdb.py` from within GDB while
+stopped. To connect GDB to the built in VS Code debugger, you can use the following
+launch configuration.
+
+```json
+{
+    "name": "Connect to GDB Server",
+    "type": "cppdbg",
+    "request": "launch",
+    "program": "${workspaceRoot}/Build/QemuSbsaPkg/DEBUG_GCC5/AARCH64/MdeModulePkg/Core/Dxe/DxeMain/DEBUG/DxeCore.debug",

```

Comment:
Might want to explicitly call out this is a AARCH QemuSbsaPkg example above so people know to change the path if using QemuQ35Pkg.

Reply:
And/or explain why it is here.